### PR TITLE
Remove wlwmanifest.xml from core

### PR DESCRIPTION
**Pull request by jdevalk**
Removes `wlwmanifest.xml` from core and all references to it.

Trac ticket: [https://core.trac.wordpress.org/ticket/41404](https://core.trac.wordpress.org/ticket/41404)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/general-template.php [hunk 1]
```diff
@@ -3372,19 +3372,6 @@ function rsd_link() {
 	);
 }
 
-/**
- * Displays the link to the Windows Live Writer manifest file.
- *
- * @link https://msdn.microsoft.com/en-us/library/bb463265.aspx
- * @since 2.3.1
- */
-function wlwmanifest_link() {
```
Comment (felixarntz) [inline src/wp-includes/general-template.php:3381]: This function needs to be deprecated rather than removed.

**Review by felixarntz — COMMENTED** [review]
@jdevalk Requesting one change here, other than that this looks good to me.

File: src/wp-includes/general-template.php (hunk 1 above)
Comment (jdevalk) [reply src/wp-includes/general-template.php:3381]: Fair point. Doing that.

Comment (jdevalk) [reply src/wp-includes/general-template.php:3381]: Done in 30014c232535caa4bc0fc55865ef9dd9c5689247

**Review by felixarntz — APPROVED** [review]
Thanks @jdevalk!

**Comment by SergeyBiryukov** [comment]
Looking at the original commit in [r6192](https://core.trac.wordpress.org/changeset/6192), it appears that the [wp-includes/images/wlw/](https://core.trac.wordpress.org/browser/tags/6.2/src/wp-includes/images/wlw) directory can also be removed. Will handle on commit :)

**Comment by SergeyBiryukov** [comment]
Thanks for the PR! Merged in [r55620](https://core.trac.wordpress.org/changeset/55620).

**Pull request closed without merging**